src/common/reducers/handlers/cards.js

```diff
@@ -116,8 +133,23 @@ function createCardFromProps(props) {
   return card;
 }
 
+function areIdenticalCards(card1, card2) {
+  // TODO: Once we have better UX for this, it's time to start getting stricter
+  // (no longer care about the name, and check abilities/command rather than text).
+  return compareCertainKeys(card1, card2, ['name', 'type', 'cost', 'text', 'stats']);
```

Comment:
Agreed, but this is kind of a placeholder method for now, and it's mainly there to make development easier for me. 

In v0.5, we'll handle this a lot better: ignore names and text, and actually replace existing matching cards rather than just ignoring them. There's just some trickiness involved because, for example, decks that have the old cards in them would have to be replaced to use the updated cards.